vnext/local-cli/runWindows/utils/build.js

```diff
@@ -83,12 +83,22 @@ async function restoreNuGetPackages(options, slnFile, verbose) {
   ensureNugetSpinner.succeed('Found NuGet Binary');
 
   const msbuildTools = MSBuildTools.findAvailableVersion('x86', verbose);
-  await nugetRestore(
-    nugetPath,
-    slnFile,
-    verbose,
-    msbuildTools.installationVersion,
-  );
+  try {
+    await nugetRestore(
+      nugetPath,
+      slnFile,
+      verbose,
+      msbuildTools.installationVersion,
+    );
+  } catch (e) {
+    if (!options.isRetryingNuget) {
+      options.isRetryingNuget = true;
+      fs.unlinkSync(nugetPath);
+      await restoreNuGetPackages(options, slnFile, verbose);
```

Comment:
Nit: this + the bottom line could be rewritten as `restoreNuGetPackages(options, slnFile, verbose);`. This would effectively append the continuation into the promise chain and let the state for the outer async function go away.